index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+  <header class="container round">
+      <nav class="toolbar">
+        <div class="logo space blue">
+          My logo
+        </div>
+        <ul class="menu liststyle">
+          <li><a class="onhover pressed" href="#">Portfolio</a></li>
+          <li><a class="onhover pressed" href="#">About</a></li>
+          <li><a class="onhover pressed" href="#">Contact</a></li>
+        </ul>
+      </nav>
+
+      <nav class="mobmenu space">
+        <div class="logo space blue">
+          My logo
+        </div>
+        <div class="">
+          <a href="#"><img src="Img/Union.svg" alt="hamburger"></a>
+        </div>
+      </nav>
+
+    <section class="headline round">
+      <div class="wide">
+        <h1 class="space">I’m Nadiia.</h1>
+        <h1 class="space">Glad to see you!</h1>
+        <p class="space">
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <br>
+        <p class="blue space">LET'S CONNECT</p>
+        <ul class="liststyle space">
+          <li><a class="onhover" href="#"><img src="Img/twitter.svg" alt="twitter"></a></li>
+          <li><a class="onhover" href="#"><img src="Img/Linkedinicon.svg" alt="linkedin"></a></li>
+          <li><a class="onhover" href="#"><img src="Img/medium.svg" alt="medium"></a></li>
+          <li><a class="onhover" href="#"><img src="Img/github.svg" alt="github"></a></li>
+          <li><a class="onhover" href="#"><img src="Img/angellist.svg" alt="angellist"></a></li>
+        </ul>
+      </div>
+
+    </section>
```

Comment:
- [ ] Headline is almost there! Let's take this `section` out of the `header` tag, it's a better practice to leave the `header` tag for the toolbar only.